grammars/jsdoc.cson

```diff
@@ -252,28 +275,29 @@
         'name': 'variable.other.jsdoc'
       }
       {
-        'name': 'variable.other.jsdoc'
         'begin': '\\['
-        'end': '\\]|(?=\\*/)'
+        'beginCaptures':
+          '0':
+            'name': 'variable.other.jsdoc'
+        'contentName': 'variable.other.jsdoc'
```

Comment:
Use `name` here instead of `contentName` if the begin and end captures should get the same scopes.